config/cordex.yaml

```diff
@@ -0,0 +1,9 @@
+builder: null
+
+translator: CordexTranslator
+
+sources:
+
+  - metadata_yaml: /g/data/xp65/admin/access-nri-intake-catalog/config/metadata_sources/cordex-ig45/metadata.yaml
```

Comment:
I don't think this is a path where we want to be storing live `YAML`... see discussion on #200